keyboards/lazydesigners/the30/readme.md

```diff
@@ -0,0 +1,17 @@
+# THE30
+
+![the30](https://i.loli.net/2019/11/06/KTn3dNjreigovRz.jpg)
+
+A 30% custom keyboard designed and produced by [LazyDesigners](http://lazydesigners.cn).
+
+Keyboard Maintainer: [Erovia](https://github.com/Erovia)  
+Hardware Supported: THE30  
+Hardware Availability: Check for GBs on [Geekhack](https://geekhack.org) and on [LazyDesigner's homepage.](http://lazydesigners.cn)
```

Comment:
```suggestion
* Hardware Availability: Check for GBs on [Geekhack](https://geekhack.org) and on [LazyDesigner's homepage.](http://lazydesigners.cn)
```

Reply:
The usage of list is a new convention?
Every keyboard I checked uses simply new lines. Even the [docs](https://docs.qmk.fm/#/documentation_templates?id=keyboard-readmemd-template).

Reply:
Yes, we changed the template in https://github.com/qmk/qmk_firmware/pull/7221

Reply:
Got it. https://github.com/qmk/qmk_firmware/blob/master/docs/documentation_templates.md will need an update, though.
(I will open a PR for it, if nobody does before me.)